documentation/usage/running-storm-on-parametric-models.md

```diff
@@ -10,108 +10,163 @@ categories: [Use Storm]
+The gd engine is based on work in {% cite HSJMK22 %}
 
 ```console
-$ storm-pars --help region
-$ storm-pars --help parametric
+$ storm-pars  --mode feasibility --feasibility:method gd --prism brp.pm  --prop 'P<=0.01 [F s=5]'
 ```
+In this case, the region for searching cannot be given.
 
-## Monotonicity analysis
+The complete output to be expected is:
 
-Storm can also check if the parameters of a model are monotonic in regard to a certain property. For this, we use the `--monotonicity-analysis` option.
+{% include includes/show_output.html class="pla_parametric_models" path="parametric/brp_feasibility_pla.out" %}
 
-### Example 3 (Checking the monotonicity of a parameter)
-Again, we use the BRP model and check if any of the parameters are monotonic:
+In particular, the output says
 
 ```console
-$ storm-pars --prism brp.pm --prop 'P=? [F s=5]' --region '0.1 <= pL <= 0.9, 0.1 <=pK <=0.9' --monotonicity-analysis --bisimulation
+Result at initial state: 0.001706273496 ( approx. 0.001706273496) at [pK=62679533432486317/72057594037927936, pL=62679533432486317/72057594037927936].
 ```
-We use `--bisimulation` to apply further model simplification and significantly reduce the workload of the model checker.
-This results in the following output:
 
-{% include includes/show_output.html class="mon_parametric_models" path="parametric/brp_mon.out" %}
+ which gives values for pK and pL such that `P<=0.01 [F s=5]`  is satisfied.
 
-If we want to take a look at the reachability order that is built during monotonicity checking, we can add the option `--mon:dotOutput` followed by an optional filename. Storm will then write a dot output of the order into the file which can then be processed using, e.g., [Graphviz](https://graphviz.org/). In this case, it results in the following graph:
+## Verification
+To prove the absence of a solution, we can use parameter lifting. A comprehensive overview on the theoretical backgrounds is given in {% cite JAHJKQV19 %}.
 
-![Reachability Order]({{ '/pics/parametric-brp-mon-dot.png' | relative_url }} "Reachability Order"){: .center-image width="300"}
+```console
+$ storm-pars  --mode verification --prism brp.pm  --prop 'P<=0.9999 [F s=5]' --region "0.1 <= pL <= 0.9, 0.1 <= pK <= 0.9"
+```
+will provide output
 
-For checking the monotonicity on samples, we use `--mon:samples` and provide it with the number of samples we want to be taken.
 
-For more options on monotonictiy checking and their usages, we can use the `--help` option:
+{% include includes/show_output.html class="pla_parametric_models" path="parametric/brp_verification_valid.out" %}
 
+In contrast, running
 ```console
-$ storm-pars --help mon
+$ storm-pars  --mode verification --prism brp.pm  --prop 'P<=0.99 [F s=5]' --region "0.1 <= pL <= 0.9, 0.1 <= pK <= 0.9"
 ```
+will yield output that says that the statement is *not* true:
+
+{% include includes/show_output.html class="pla_parametric_models" path="parametric/brp_verification_invalid.out" %}
+
```

Comment:
Maybe say that feasibility mode can then be used to find a violating assignment. But I guess this should also be clear by now.